Wrappers/Python/ccpi/optimisation/operators/IdentityOperator.py

```diff
@@ -35,7 +35,7 @@ def adjoint(self,x, out=None):
         else:
             out.fill(x)
         
-    def norm(self):
+    def calculate_norm(self):
```

Comment:
Here it makes sense with a dedicted `calculate_norm`, as there is no need to call the power method.